charts/telepresence/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "telepresence.fullname" . }}
+  labels:
+    {{- include "telepresence.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "telepresence.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "telepresence.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
```

Comment:
We also have the following environment variables: 
```
        - name: LOG_LEVEL
          value: debug
        - name: SYSTEMA_HOST
          value: app.getambassador.io
        - name: SYSTEMA_PORT
          value: "443"
        - name: CLUSTER_ID
          value: <uid from default namespace>
```
LOG_LEVEL we should definitely add (use debug as the default), the middle ones are already the default so that's not a huge deal.  The last one you can get from `telepresence current-cluster-id` or `kubectl get ns default -o yaml` which I think we should put in the docs for now.. I want to deprecate it as an env var once we move the installer into the traffic-manager (and the traffic-manager can actually talk to the k8s cluster and get this info itself).. but for now I say we add it to the chart and add a comment above the value about how the user can get it.

Reply:
Thanks! I did not see these in `k8s/manager.yaml` so I omitted them.

I have added the `--set clusterID=$(kubectl get ns default -o jsonpath='{.metadata.uid'})` to the README